src/views/Observatory.vue

```diff
@@ -42,29 +44,37 @@
 <script>
 import Container from "@/components/Container";
 import GradientText from "@/components/GradientText";
-import CustomLink from "../components/CustomLink";
-// import Player from "@vimeo/player";
-
+import CustomLink from "@/components/CustomLink";
+import VideoTemplate from "@/components/VideoTemplate";
+import observatoryData from "../observatoryData";
 export default {
   name: "Observatory",
   components: {
     Container,
     CustomLink,
     GradientText,
+    VideoTemplate,
   },
-  /* -- Allows to listen to on video end event
-  mounted() {
-    const screenWidth = window.innerWidth;
-    let videoOptions = {
-      id: "https://vimeo.com/82886396",
-      width: screenWidth,
-      background: 1,
+  data() {
```

Comment:
Rather than this, why not find the observatory by its ID from the route on load and just set that one to the data so we dont have to keep looking it up at its index to grab data from it

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find